core/src/execution/datafusion/spark_hash.rs

```diff
@@ -99,12 +98,144 @@ pub(crate) fn spark_compatible_murmur3_hash<T: AsRef<[u8]>>(data: T, seed: u32)
     }
 }
 
+const CHUNK_SIZE: usize = 32;
+
+pub const PRIME_1: u64 = 11_400_714_785_074_694_791;
+pub const PRIME_2: u64 = 14_029_467_366_897_019_727;
+pub const PRIME_3: u64 = 1_609_587_929_392_839_161;
+pub const PRIME_4: u64 = 9_650_029_242_287_828_579;
+pub const PRIME_5: u64 = 2_870_177_450_012_600_261;
```

Comment:
Thanks for the review @advancedxy. I moved xxhash64 into it's own file under `execution.datafusion.expressions` which makes sense also because this is a regular SQL function that users can call.